coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/subaward/ProposalBudgetSubawardsPage.xml

```diff
@@ -138,7 +141,7 @@
 					p:order="10" />
 				<bean parent="PropBudget-SubawardsAttachments-FileUpload-Section" 
 					p:hidden="@{dialogDataObject?.budgetSubAwardFiles == null ? false : !dialogDataObject.budgetSubAwardFiles.isEmpty()}" 
-					p:order="20" />
+					p:order="20" p:render="@{!#parent.readOnly}"/>
```

Comment:
This section is hidden unless the user clicks a button(which will also be hidden), so not rendering it in this case seemed better. As far as a standard, no. But I prefer to show buttons as disabled instead of hiding them as it gives a better user experience. Instead of wondering how to do something you can't, you simply see that you can't do it.